app/assets/javascripts/surveys.js

```diff
@@ -3,39 +3,42 @@
 (function ($) {
   'use strict'
   window.GOVUK = window.GOVUK || {}
+  var cancelButton = '<a href="#email-survey-cancel" aria-labelledby="feedback-heading email-survey-cancel" id="email-survey-cancel" role="button" class="close-button">Close</a>'
```

Comment:
When testing this in a screenreader we found that a link that says 'no thanks' without context is hard to understand.

With these updates the link will now read out the title followed with 'close button' which makes sense on it's own